lib/internal/test_runner/test.js

```diff
@@ -560,7 +561,7 @@ class Test extends AsyncResource {
         const map = lazyFindSourceMap(this.loc.file);
         const entry = map?.findEntry(this.loc.line - 1, this.loc.column - 1);
 
-        if (entry !== undefined) {
+        if (entry !== undefined && ObjectKeys(entry).length > 0) {
```

Comment:
I feel like it might be faster to check that the entry contains an `originalSource` value.

Reply:
It makes sense given that the `findEntry` return is either a valid object (with all props) or an empty one.